configure.ac

```diff
@@ -79,7 +79,7 @@ AC_ARG_ENABLE(ipv6only, AS_HELP_STRING([--enable-ipv6only],
               [Build IPv6-only (default: no)]),
               [], [enable_ipv6only=no])
 AC_ARG_ENABLE(kerberos, AS_HELP_STRING([--enable-kerberos],
-              [Build kerberos support (default: no)]),
+              [Build kerberos support (prefer PAM if available) (default: no)]),
```

Comment:
Good points - thanks.

There are no docs I'm aware of. To answer #1715 I had to dig in to the source code.

I've addressed both of your other points:-
- I've made the recommendation to use PAM more explicit.
- I've added code so that at most one of `--enable-pam`, `--enable-bsd`, `--enable-kerberos` and -`-enable-pamuserpass` can be specified - this is what the logic in `sesman/Makefile.am` is effectively doing without checking. The config log now says which auth mech is built in to sesman.